bots/README.md

```diff
@@ -31,7 +31,7 @@ For more details, see the [Bots Architecture section](bots-architecture/#bot-pla
 
 These are the basic steps for using bots with Rocket.Chat:
 
-1. A bot user is [created by an admin](creating-bot-users/) on the server;
+1. A bot user is [created by an admin](create-and-run-a-bot/) on the server;
```

Comment:
Please change in the "Where are bots hosted?" section...
"Rocket.Chat supports bots that are hosted outside of your Rocket.Chat instance"

to

"Like most chatbot platforms,  Rocket.Chat support bots that are hosted outside of your Rocket.Chat instance, running on an external server."

Reply:
Fixed